pkg/syncthing/process.go

```diff
@@ -16,22 +14,47 @@
 package syncthing
 
 import (
-	"os"
-	"strings"
+	"time"
+
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/shirou/gopsutil/process"
 )
 
-func terminate(pid int, wait bool) error {
-	proc := os.Process{Pid: pid}
-	if err := proc.Signal(os.Interrupt); err != nil {
-		if strings.Contains(err.Error(), "process already finished") {
-			return nil
-		}
+func getParentExe(p *process.Process) string {
+	parent, err := p.Parent()
+	if err != nil {
+		log.Infof("error getting parent process: %s", err.Error())
+		return ""
+	}
+	if parent == nil {
+		return ""
+	}
+	parentExe, err := parent.Exe()
+	if err != nil {
+		log.Infof("error getting parent process exe: %s", err.Error())
+		return ""
+	}
+	return parentExe
+
+}
 
+func terminate(p *process.Process, wait bool) error {
+	if err := p.Terminate(); err != nil {
 		return err
 	}
 
 	if wait {
-		defer proc.Wait() // nolint: errcheck
+		isRunning, err := p.IsRunning()
```

Comment:
double check, but I believe p.IsRunning doesn't work as expected in windows if the process doesn't exist.